src/lib/actions/pannable.ts

```diff
@@ -156,9 +165,20 @@ export function drag(
   }
 
   return (event) => {
-    coords.update(($coords) => ({
-      x: bounded($coords.x + (axis !== 'y' ? event.detail.dx : 0), 'x'),
-      y: bounded($coords.y + (axis !== 'x' ? event.detail.dy : 0), 'y'),
-    }));
+    coords.update(($coords) => {
+      //if the coords are just one number, we check how to interpret this
+      if(typeof $coords === "number"){
+        if(axis==="y"){
```

Comment:
Let's make sure to give useful feedback if someone specified `'xy'` as axis. For example:

```suggestion
if (axis === 'xy')
      throw new Error('Please specify an appropriate axis (x or y)');
if(axis==="y"){
```